<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadingDialog</class>
 <widget class="QDialog" name="ShadingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxVH">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>91</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>方向选择</string>
   </property>
   <widget class="QRadioButton" name="radioButtonXY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>XY</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonX">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>33</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonY">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>33</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxGainDark">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>444</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>暗缺陷增益</string>
   </property>
   <widget class="QSlider" name="horizontalSliderGainDark">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>30</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGainDark">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>19</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddGainDark">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubGainDark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDir">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>241</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>缺陷提取选择</string>
   </property>
   <widget class="QRadioButton" name="radioButtonB">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>明</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonD">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>暗</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBD">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>明暗</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBDRand">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>明暗个别</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMiddleValue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>304</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>平均值补正</string>
   </property>
   <widget class="QLineEdit" name="txtMiddleValue">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>21</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxTh">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>301</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>排除干扰值</string>
   </property>
   <widget class="QSlider" name="horizontalSliderTH">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>33</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTh">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>16</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddTH">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>33</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubTH">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxGain">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>372</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>缺陷增益</string>
   </property>
   <widget class="QSlider" name="horizontalSliderGain">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>31</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGain">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>13</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddGain">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>31</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubGain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMode">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>151</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>模式选择</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAverage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>平均值补正</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMiddle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>中间值补正</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonShade">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>阴影补正</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxScale">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>11</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>窗口大小</string>
   </property>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>28</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3*3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>26</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5*5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>26</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7*7</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>9*9</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>56</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>11*11</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>56</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>13*13</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
